node/test/go/dev-mode/test/fixtures/lambdas/with-internal.js

```diff
@@ -2,5 +2,7 @@
 
 module.exports.handler = (event, context, callback) => {
   console.log('with-internal 1');
+  console.warn('with-internal warning 1');
+  console.error(new Error('with-internal warning 1'));
```

Comment:
just a message confusion, as it should probably be `with-internal error..`

Reply:
```suggestion
  console.error(new Error('with-internal error 1'));
```

Ahh yes good call 👍